processing/src/main/java/org/apache/druid/frame/file/FrameFile.java

```diff
@@ -186,7 +186,7 @@ static FrameFile open(
       // Verify minimum file length.
       if (fileLength <
           FrameFileWriter.MAGIC.length + FrameFileWriter.TRAILER_LENGTH + Byte.BYTES /* MARKER_NO_MORE_FRAMES */) {
-        throw new IOE("File [%s] is too short (size = [%,d])", file, fileLength);
+        throw new IOE("File[%s] is too short (size = [%,d])", file, fileLength);
```

Comment:
In all of these places, should we throw `DruidException` instead, since these are meant to be user-facing errors?

Reply:
Hmm, I can't really think of a good admin, user, or operator oriented message for this condition. It probably indicates some bug or data corruption. They probably all need to be "developer" messages. Although I think "developer" DruidExceptions get wrapped around random exceptions automatically… perhaps they're ok the way they are? Please LMK if you had something else in mind though.